modules/customize-certificates-api-add-named.adoc

```diff
@@ -113,3 +113,8 @@ kube-apiserver   {product-version}.0     True        False         False      14
 ----
 +
 If `PROGRESSING` is showing `True`, wait a few minutes and try again.
+
+[NOTE]
+====
+The new revision of the Kubernetes API server will only roll out if the API Server named certificate is added for the first time. At the time of renewing the API Server named certificate the new revision of the Kubernetes API server will not roll out, as the kube-apiserver pods dynamically reload the updated certificate.
```

Comment:
```suggestion
The new revision of the Kubernetes API server will only roll out if the API Server named certificate is added for the first time. At the time of renewing the API Server named certificate, the new revision of the Kubernetes API server will not roll out, as the `kube-apiserver` pods dynamically reload the updated certificate.
```

Reply:
A couple suggestions here:
- A comma is needed after the phrase "At the time of renewing..."
- Backticks are needed around `kube-apiserver` since they're used earlier on the page.